Week3/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
```

Comment:
The package.json file is the greeting card of your project. I can contain your name, your contact information like website (I mean, it is a good moment to make some networking with your great project, right?!), also some description of your project, scripts to help people running your application, and etc. Make good use of this file. =)

Here is a link for more info on it: https://docs.npmjs.com/files/package.json